packages/design-system-docs/src/pages/components/DatePicker/_CalendarPickerDateField.docs.scss

```diff
@@ -0,0 +1,50 @@
+/*
+Date field (with calendar picker)
+
+@uswds https://designsystem.digital.gov/components/date-picker/
+
+Date field (with calendar picker), also known as _date picker_, provides a single date entry with a text field or selection through a visual calendar element tied to the input.
+
+Style guide: components.date-picker
+*/
+
+/*
+`<SingleInputDateField>`
+
+@react-example DatePicker.example.jsx
+
+@react-props SingleInputDateField.tsx
+
+Style guide: components.date-picker.react
+*/
+
+/*
+---
+
+### When to use
+
+- Use the date picker for dates that have happened within the last year, in the future, or is relevant to selecting the day of the week.
+
+### When to consider alternatives
+
+- If users are entering a memorable date. Instead, use either single- or multi-input date fields.
```

Comment:
might be helpful to turn `single-` and `multi-input` into links to those component pages